modules/disk.sh

```diff
@@ -114,21 +114,27 @@ disk_create_file_system_from_folder() {
 
   run_and_log_cmd "dd if=/dev/zero of=${2} seek=${3} count=0 bs=512 status=none"
 
+  case ${1} in
+      *data/ ) EXTRA_OPTS="${MENDER_DATA_PART_MKFS_OPTS}";;
+      *rootfs/ ) EXTRA_OPTS="${MENDER_ROOT_PART_MKFS_OPTS}";;
+      * ) EXTRA_OPTS="";;
+  esac
+
   case "$4" in
     "ext4")
       MKFS_EXT4="/usr/bin/mkfs.ext4"
       if [ ! -f ${MKFS_EXT4} ]; then
         MKFS_EXT4="/sbin/mkfs.ext4"
       fi
-      run_and_log_cmd "${MKFS_EXT4} -q -F ${2}"
+      run_and_log_cmd "${MKFS_EXT4} -q -F ${2} -i 4096 ${EXTRA_OPTS}"
```

Comment:
The `-i 4096` was snuck into the same commit. Why? If it's intentional I think this should be in a separate commit.

Reply:
Rebase failure.  It's fixed now